EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -237,32 +238,49 @@ void V2World::importDefaultPops()
 
 	set<string> filenames;
 	Utils::GetAllFilesInFolder("./blankMod/output/history/pops/1836.1.1/", filenames);
+
+	LOG(LogLevel::Info) << "Parsing minority pops mappings";
+
+	std::ifstream minPopFile("minorityPops.txt");
+	if (minPopFile.fail())
+	{
+		LOG(LogLevel::Error) << "Could not parse file minorityPops.txt";

```

Comment:
Unless you catch the exception yourself, the top-level try/catch block should log it. So you don't need this line.

Reply:
dropped.